dawn/main/ansible/Ansible.js

```diff
@@ -117,62 +44,212 @@ function buildProto(data) {
     });
     return GamepadMsg;
   });
+
   const message = new DawnData({
     student_code_status: status,
     gamepads,
   });
+
   return message;
 }
 
-/*
- * IPC Connection with sagas.js' ansibleGamepads()
- * Sends messages when Gamepad information changes
- * or when 100 ms has passed (with 50 ms cooldown)
- */
-ipcMain.on('stateUpdate', (event, data) => {
-  const message = buildProto(data).encode().toBuffer();
-  console.log('Dawn Sent UDP');
-  sendSocket.send(message, sendPort, runtimeIP, (err) => {
-    if (err) {
-      console.error('UDP Sending Error');
+const LISTEN_PORT = 1235;
```

Comment:
move these constants to the top of the file next to each other. 

easily accessible, visible, modifiable.